WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -98,6 +104,13 @@
     private Debouncer mDebouncer = new Debouncer();
     private SitePickerViewModel mViewModel;
 
+    private HashSet<Integer> mHashSelectedPositions = new HashSet<>();
```

Comment:
Hi @hafizrahman Nitpick 😅  what do you think about renaming this to `mSelectedPositions` for consistency with all the other variables since they don't contain any identifier of their underlying type?

Reply:
That makes a lot of sense! I did not notice the no-type in variable name rule before, thanks.